include/circt/Dialect/FIRRTL/Passes.h

```diff
@@ -56,8 +56,6 @@ std::unique_ptr<mlir::Pass> createLowerFIRRTLTypesPass(
     PreserveAggregate::PreserveMode mode = PreserveAggregate::None,
     PreserveAggregate::PreserveMode memoryMode = PreserveAggregate::None);
 
-std::unique_ptr<mlir::Pass> createLowerBundleVectorTypesPass();
-
```

Comment:
Why delete it

Reply:
The function was added in e5246fe83c6f421b0b8cfe42a542f74d55638c6b but renamed to `createLowerFIRRTLTypesPass` in 4f7c50791e90b70d4b1f3814f47215265ce39205, and the declaration was forgotten. So it's a dead invalid declaration.